R/methods-XCMSnExp.R

```diff
@@ -361,7 +361,7 @@ setMethod("chromPeaks", "XCMSnExp", function(object, bySample = FALSE,
                    drop = FALSE]
     ## Select peaks within rt range.
     if (length(rt)) {
-        rt <- range(rt)
+        rt <- range(as.numeric(rt))
```

Comment:
Why was that conversion needed ? What "broken" rt did you observe ?

Reply:
this is just a failsafe - we had cases in which a `character` was passed with `rt` - and the function did neither throw an error nor show a warning.